unix/src/machine_stats/modules/proc_stats.py

```diff
@@ -115,6 +116,15 @@ def parse_status(process_path):
         stats["name"] = status["name"]
         stats["path"] = "/"
 
+    # Finally, let's see if we can read the username
+    if status.get("uid"):
+        try:
+            uid = int(status["uid"].split()[0])
+            user_entry = getpwuid(uid)
+            stats["user"] = user_entry.pw_name
+        except KeyError as e:
```

Comment:
```suggestion
        except KeyError:
```
There's no need to capture the exception as a variable `e`, since we're not going to use it.

Reply:
Ahh thanks for that.